lib/test/unit/capybara.rb

```diff
@@ -66,22 +66,29 @@ def initialize(node, kind, locator, message)
 
     # @private
     module FindErrorWrapper
-      def find(*args)
-        begin
+      if ::Capybara::VERSION >= "3.0.0.rc1"
+        def find(*args, **options, &optional_filter_block)
+          super(*args, options, &optional_filter_block)
```

Comment:
maybe `super` don't need argument's.
I am going to have a new commit to fix this...